include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -39,9 +39,8 @@
 #if defined(__LIBSYCL_MAJOR_VERSION) && defined(__LIBSYCL_MINOR_VERSION) && defined(__LIBSYCL_PATCH_VERSION)
 #    define _ONEDPL_LIBSYCL_VERSION                                                                                    \
         (__LIBSYCL_MAJOR_VERSION * 10000 + __LIBSYCL_MINOR_VERSION * 100 + __LIBSYCL_PATCH_VERSION)
-#else
-#    define _ONEDPL_LIBSYCL_VERSION 0
 #endif
+#define _ONEDPL_LIBSYCL_VERSION_LESS_THAN(_VERSION) (_ONEDPL_LIBSYCL_VERSION && _ONEDPL_LIBSYCL_VERSION < _VERSION)
```

Comment:
I propose to define instead of this macro or as additional macros something like
`_ONEDPL_LIBSYCL_VERSION_GREAT_EQ_THAN`

The goal - to avoid constructions like
`#define _ONEDPL_SYCL2020_KNOWN_IDENTITY_PRESENT (!_ONEDPL_LIBSYCL_VERSION_LESS_THAN(50300))`
(I mean the negotiation).

Reply:
Unfortunately, "libsycl-greater-or-equal" only applies to the libsycl library, while "not libsycl-less-than" also covers all other libraries.

Reply:
May be.
But from my point of view this negotiation make defines a little bit heavy for uderstanding.

Reply:
The comment above the block of macros is added to clarify the logic.